classroom-dashboard/app/controllers/students_controller.rb

```diff
@@ -1,7 +1,8 @@
 class StudentsController < ApplicationController
   before_action :find_student, only: [:edit, :show, :update, :destroy]
   before_action :find_section
-
+  
+  # Do empty methods need to be filled?
```

Comment:
Empty methods are good, they're still running find_student as written on line 2.

However, after creating a student, the user is redirected to the student list page (section_students_path(@section)) but after editing a student, the user is redirected to section page (section_path(@section)). We should decide which we want and comment it appropriately. 

Everything else looks good! It'd be awesome if our parameters firstName and lastname were consistently formatted (either both camel case or neither). It would probably take a migration to fix that; not sure what priority it is?